software_reseller/data/product_template.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="__export__.product_template_1_f06ecb8a" model="product.template" context="{'studio': True, 'create_product_product': False}">
```

Comment:
products are probably more demo data than data.

Reply:
@pimodoo As per this task https://www.odoo.com/web#id=3446301&cids=2&menu_id=4720&action=4043&model=project.task&view_type=form Product must be as data.

so can you re-confirm it?

Reply:
Ok, so we can keep it in data